py/LICENSE

```diff
@@ -0,0 +1,104 @@
+Business Source License 1.1
```

Comment:
I think we should use MIT or even public domain otherwise we will not be able to ship with SDKs. Also in the case of BSL we need @BYK to setup his stuff for updating Change date.

The license could potentially also apply to the whole repo

Reply:
I think you are right, this should not be BSL.

Reply:
Actually thinking over this again, I suspect it may be safer to start with BSL as it is more restrictive, then we can open it up if/when we need. Restricting it later would be harder.
(I can add the automation to change the date, I figured out how it works for Snuba).
What do you think ?

Reply:
I can't ship BSL in the SDK in any capacity, it will be a hard blocker. Asking `#team-oss` in Slack.

Reply:
MIT it is as discussed over at Slack.

Reply:
done